sycl/test-e2e/Matrix/joint_matrix_prefetch.cpp

```diff
@@ -6,11 +6,12 @@
 //
 //===----------------------------------------------------------------------===//
 // REQUIRES: aspect-ext_intel_matrix
+
+// UNSUPPORTED: gpu-intel-dg2
+
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
-// XFAIL: gpu
```

Comment:
This test is still failing with intel/llvm compiler on PVC, so removing this xfail will cause failure in CI